src/workspaces/index.ts

```diff
@@ -133,7 +133,13 @@ export class Workspaces {
 
   public editWorkspace(
     workspaceId: string,
-    details: { name?: string; description?: string; address?: string },
+    details: {
+      name?: string;
+      description?: string;
+      address?: string;
+      phoneNumber?: string | null;
+      email?: string | null
```

Comment:
The email has its own process to be edited, have you checked this new way is correct?